fae.gemspec

```diff
@@ -26,12 +26,12 @@ Gem::Specification.new do |s|
   s.add_dependency 'simple_form', '>= 3.1.0'
   s.add_dependency 'jquery-ui-rails', '~> 4.2.1'
   s.add_dependency 'remotipart', '~> 1.3.1'
-  s.add_dependency 'carrierwave', '~> 0.10.0'
+  s.add_dependency 'carrierwave', '~> 1.0'
```

Comment:
I will be updating `carrierwave` in v2.0, which is releasing soon. Please revert this in this PR.